apps/phone/src/apps/contacts/hooks/useModal.tsx

```diff
@@ -0,0 +1,8 @@
+import { useRecoilState } from 'recoil';
+import { showMoneyModal } from './state';
+
+export const useModal = () => {
+  const [contactPayModal, setcontactPayModal] = useRecoilState<boolean>(showMoneyModal);
```

Comment:
nit: this should be setContactPayModal to keep consistent with the rest of the project

Reply:
changed